website/src/content/docs/docs/howtos/Generate client libraries/10versioning.mdx

```diff
@@ -464,6 +464,178 @@ func (client *ServiceClient) StableFunctionality(ctx context.Context, stableMode
 
 </ClientTabs>
 
+### New model/operation added in new api version
+
+You can use `@added` decorator to indicate when a specific Api feature was first introduced.
+When applied to models, operations, or parameters, `@added` marks the Api Version at which these elements became available. This prevents confusing error messages for customers using older version clients.
+
+#### New operation added in new api version
+
+In this example, `stableFunctionality` is marked with `@added(Versions.v2023_11_02)`, meaning this operation only becomes available starting from the `2023-11-02` version. Elements without the `@added` decorator, like `previewFunctionality`, are available across all defined versions.
+
+<ClientTabs>
+
+<ClientTabItem lang="typespec">
+
+```typespec
+import "@typespec/versioning";
+import "@typespec/http";
+
+using Versioning;
+using Http;
+
+@versioned(Versions)
+@service
+namespace My.Service;
+
+enum Versions {
+  v2023_11_01_preview: "2023-11-01-preview",
+  v2023_11_02: "2023-11-02",
```

Comment:
This does not match the schema of `@previewVersion`?

Reply:
I don't want to introduce `@previewVersion` in this section so let me update the preview api version to another stable version.